.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,24 +1,43 @@
 ---
 name: Bug report
-about: File a bug report in OJS, OMP or OPS. Try our support forum linked below if you can't provide reproduction steps and technical specifications.
-title: ''
+about: File a bug report in OJS, OMP or OPS. Try our Software Support forum, linked below, if you cannot provide reproducible steps and technical specifications.
+title: '[Main Feature] | Workflow Section - Page, Description of bug'
 labels: ''
 assignees: ''
 
 ---
 
-**Describe the bug**
-Please tell us what happens, what you expected to happen, and why you think it is a bug in the software.
+**Description:**
+A detailed explanation of the bug, include any unusual behavior or unique details.  This is the chance to provide some context surrounding the bug.
```

Comment:
Consider being more specific about "context surrounding the bug", or remove this line.